WordPress/Resources/release_notes.txt

```diff
@@ -1,7 +1,9 @@
-New year, new look! We updated comment threads with a shiny new design for author labels, accessory and share buttons, and row and reply highlighting. We also made a few updates to the Block Editor.
+* [**] Video uploads: video upload is now limited to 5 minutes per video on free plans.
+* [*] Block editor: Give multi-line block names central alignment in inserter [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4343]
+* [**] Block editor: Fix missing translations by refactoring the editor initialization code [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4332]
+* [**] Block editor: Add Jetpack and Layout Grid translations [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4359]
+* [**] Block editor: Fix text formatting mode lost after backspace is used [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4423]
+* [*] Block editor: Add missing translations of unsupported block editor modal [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4410]
+
+* [**] Time zone suggester: we have a new time zone selection screen that suggests the time zone based on the device, and improves search.
```

Comment:
Speaking of pdnsEh-6h-p2 you shoiuld copy this file to `WordPress/Jetpack/Resources/release_notes.txt`.

This would both ensure that we don't forget to update the file, but also be less confusing when we point our freelancer to those files to translate release notes for WP and JP (even if this time around they might be the exact same, that won't always be the case)